Money_Counter_JB.py

```diff
@@ -0,0 +1,44 @@
+# This programme counts British coins based on their weight
+
+# Started with dictionary of UK coin weights
+
+coindict = {
+    "1": 3.56,
+    "2": 7.12,
+    "5": 3.25,
+    "10": 6.5,
+    "20": 5.0,
+    "50": 8.0
+    }
+
```

Comment:
Before the logic, you could set some variables to store the options as an easier-to-remember name, and then you can use this elsewhere
```suggestion
weigh_coins_option = 'a'
fill_bag_option = 'b'
```